### PR TITLE
Img optimize webpack loader

### PR DESCRIPTION
**Pull request by PeteCoward**
Using https://github.com/tcoopman/image-webpack-loader to optimize images in the build process

https://github.com/tcoopman/image-webpack-loader

This does not inline svgs like we had before so the number of files spat out by the build process increases, but the total size of files is a 165 KiB win.

canoe.js     564 KiB -> 287 KiB
dist/            981 KiB  -> 816 KB

### CODE REVIEW

**Comment by PeteCoward** [comment]
I could not work out ho to configre the precision of the svg compression :)

**Comment by sergioestevess** [comment]
The available options for SVG compression: https://github.com/svg/svgo#what-it-can-do

**Comment by boerwastaken** [comment]
I've noticed that yes, one can remove redundancy and increase entropy in a preprocessing step, but then since the HTTP transport compression has more entropy to deal with, the gains are much less on the compressed size-on-wire than on the uncompressed size-on-disk. Of course a dedicated SVG minifier can do smarter optimizations, and a bunch of lossy transformations to boot, but `gzip -9` is pretty good as well (and you can also think of configuring Brotli compression, as most web browsers now support that too).

In short, if you're trying to minimize transfer size, don't compare uncompressed sizes, but `gzip -9` the artefacts, then compare, and weigh the (much smaller) savings against the baggage of more dependencies and more complicated build chains. Perhaps also do a brotli compression run and then we might want to make a case to enable brotli transport compression on our web servers.

Storage size (in SW cache) is another story of course, as I suspect the responses are stored uncompressed.

**Comment by PeteCoward** [comment]
deployed here https://asteroid-dev.temp.build/

> The available options for SVG compression: https://github.com/svg/svgo#what-it-can-do

yeah, I tried but could not find the right options sauce to see any difference

**Comment by katevoss** [comment]
How do you recommend testing and reviewing this PR @PeteCoward  ?

**Comment by PeteCoward** [comment]
Closing this as I'm not happy that it removes inlining. If we can find a way to optimize and inline that would be best.
